docfx.json

```diff
@@ -0,0 +1,41 @@
+{
+  "metadata": [
+    {
+      "src": [
+        {
+          "files": [ "**/*.sln", "**/*.cs" ],
+          "exclude": [ "**/bin/**", "**/obj/**" ],
+          "src": "Assets/MixedRealityToolkit"
```

Comment:
Should we not include other folders for generated docs (e.g. MixedRealityToolkit.Providers, MixedRealityToolkit.SDK or MixedRealityToolkit.Services)

Reply:
included all source folders for now